functions/postgre-sql-python/handlers/handler.py

```diff
@@ -3,12 +3,11 @@
 import os
 import logging
 
-PG_HOST=os.environ.get('PG_HOST')
-PG_USER=os.environ.get('PG_USER')
-PG_DATABASE=os.environ.get('PG_DATABASE')
-PG_PASSWORD=os.environ.get('PG_PASSWORD')
-PG_PORT=os.environ.get('PG_PORT')
-PG_SSL_ROOT_CERT=os.environ.get('PG_SSL_ROOT_CERT')
+PG_HOST=os.getenv('PG_HOST')
+PG_USER=os.getenv('PG_USER')
+PG_DATABASE=os.getenv('PG_DATABASE')
+PG_PASSWORD=os.getenv('PG_PASSWORD')
+PG_PORT=os.getenv('PG_PORT')
```

Comment:
:p will recycle my comment from the last PR

I don't like `os.getenv()` in this context (for required en vars) because it will silently fail if the env var is undefined. 

```suggestion
PG_HOST=os.environ["PG_HOST"]
PG_USER=os.environ["PG_USER"]
PG_DATABASE=os.environ["PG_DATABASE"]
PG_PASSWORD=os.environ["PG_PASSWORD"]
PG_PORT=os.environ["PG_PORT"]
```

Reply:
Done.